ui/src/components/sermon-notes/SermonNotesContainer.js

```diff
@@ -50,7 +53,7 @@ const SermonNotesContainer = (props) => {
         const { data, status } = await axios.put(
           '/api/user-sermon-notes/update',
           {
-            userId: user.id,
+            userId: user?.id || '',
```

Comment:
Is there a scenario when the user-sermon-notes doesn't contain userId?

Reply:
yes, that function is called on mount. But if the user is guest or no account, they would not have user object, n if we pass null to the API it'll crash lol. better give a dummy string